Add naive and chunk review files (#35)
datalater/hamancho_engstudy: review/chunk-review-jhi.md

---

### 20190412 식당운영 생각 중

## Original

### Group

> * **A** Arman, can you pleas e/ make 12 copies / of this ? /

Reviewer: ```suggestion
* **A** Arman, can you please / make 12 copies / of this ? /
```

Please make this typo correct.

> * **B** Ok,/ don't worry./ Is this for/ the meeting? /

Reviewer: ```suggestion
* **B** OK,/ don't worry./ Is this for/ the meeting? /
```

`OK` is the right one. Because it is a abbreviation for `okay`.

* **A** Yes. / well,/  we have 15 minutes / before the meeting./
* **B** Nice. / I have something / to tell you./  I'm thinking of/  running my own restaurant./
* **A** What? / Are you serious? / Why don't you / give it a second thought?/
* **B** I've alomost / made up my mind./
*
### Quest

* **A** 아르만 / 할 수 있니 / 12부 복사하다 / 이것을? /
* **B** 그럼 / 걱정하지마 / ~을 위한거니? / 회의 /
* **A** 응. / 음 / 15분 남았다 / 회의 시작하기 전에 /
* **B** 좋네 / 나 가지고 있다 / 너에게 말하다 / 나는 생각 중이다 / 내 식당을 운영하다 /
* **A** 뭐? / 농담이지? / ~하는게 어때? / 두 번 생각하다
* **B** 나 거의 했다 / 결심하다 /

## Customized

### Group

* **A** Jay, can you please / make 3 copies of / this poster?

> * **B** Ok,/  don't worry. / Is this for / the disply of your room?/  

Reviewer: Same as above.

> * **B** Ok,/  don't worry. / Is this for / the disply of your room?/  

Reviewer: ```suggestion
* **B** Ok,/  don't worry. / Is this for / the display of your room?/  
```

display (o)
